openqaoa/problems/problem.py

```diff
@@ -234,32 +243,23 @@ def coordinates(self, input_coordinates):
 
             for each_value in each_entry:
                 if not isinstance(each_value, float) and not isinstance(
-                    each_value, int
+                        each_value, int
                 ):
                     raise TypeError("The coordinates must be of type float or int")
 
         self._coordinates = np.array(input_coordinates)
 
     @staticmethod
-    def random_instance(**kwargs):
+    def random_instance(problem_size):
         """
         Creates a random instance of the Traveling Salesman problem.
 
-        Parameters
-        ----------
-        n_cities: int
-            The number of cities in the TSP instance. This is a required 
-            keyword argument.
-
         Returns
         -------
             A random instance of the Traveling Salesman problem.
         """
-        n_cities = check_kwargs(["n_cities"], [None], **kwargs)[0]
-        seed = kwargs.get("seed")
-
-        if isinstance(seed, int):
-            np.random.seed(seed)
+        np.random.seed(SEED)
```

Comment:
Do you know if doing the seed this way might cause problems. i.e. if the user wants to set his own numpy seed before running this function. Also does repeatedly running random_instance create the same instance all the time. Looks like with this setup it might.

Reply:
Yes, I have pointed this out with @vishal-ph and apparently they want this behaviour for reproducibility.